packer/ansible/roles/isc-dhcp-server/files/systemd-isc-dhcp-server

```diff
@@ -18,4 +18,5 @@
 
 # On what interfaces should the DHCP server (dhcpd) serve DHCP requests?
 #	Separate multiple interfaces with spaces, e.g. "eth0 eth1".
-INTERFACES="enp0s8"
+#   for 16.04, either ens33(e1000) or ens192(vmxnet3)
+INTERFACES="ens33 ens192"
```

Comment:
@cgx027 , I've finally use a "smart" secondary NIC name function to do what you proposed.